fiboa_cli/convert_gml.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from fiboa_cli import log
+import geopandas
+
+
+def gml_assure_columns(data, path, uri, layer, **kwargs):
+    # if GDAL opens a GML file, it generates a gfs file in which it tries to 'guess' a
+    # mapping from the GML XML file to features. This is not always correct.
+    # Call this function to add additional attributes from the GML
+    # We modify the GFS file, a more elegant solution is preferred
+    # See https://gdal.org/en/latest/drivers/vector/gml.html#schema for more info
```

Comment:
I was wondering whether it makes more sense and is a bit cleaner to just ship pre-defined GFS files for the datasets?

Reply:
I was wondering the same thing. I actually worked on a project that included a GFS file. After every base-data format change I was always struggling to update the GFS file. It took me a while to figure out this GFS file is generated. But I could document that too.

Reply:
Okay, we can work on this in a separate PR.

Reply:
See #104